awesome.js

```diff
@@ -356,9 +541,29 @@ class Awesome{
 
         /**
          * mergeDataset merges element's dataset to current default dataset of document
+         *
+         * @example
+         *
+         * defaultElementDataset = {
+         *  	property1: 'one',
+         *  	property2: 'two'
+         * }
+         *
+         * function yourElementAttributeChangeHandler(changedDataset){
+         * 		mergeDataset(myElement, changedDataset);
+         * }
+         *
+         * //after the dataset changes it will be
+         *
+         * //ElementDataset
+         *  {
+         *  	property1 : 'newProp1',
+         *  	property2 : 'newProp2'
+         *  }
+         *
```

Comment:
This example is incorrect. if the attributes change, it means the dataset already has the latest.

I think you mixed this a little with ` updateAttributesFromData `

yourElementAttributeChangeHandler should probably be componentCreatedCallback